src/shared/apis/filteredExternQuery.test.ts

```diff
@@ -1,300 +0,0 @@
-import {mocked} from 'ts-jest/utils'
```

Comment:
why are we removing this test?

Reply:
all the `filteredExternXXXX.test.ts` were copy of the original tests intended to run with `filterExtern` flag turned on. So, I have removed these tests and copied their contents to original tests.